qiskit_optimization/applications/ising/graph_application.py

```diff
@@ -0,0 +1,27 @@
+from typing import Optional
+from abc import abstractmethod
+
+import networkx as nx
+
+from .base_application import BaseApplication
+
+
+class GraphApplication(BaseApplication):
```

Comment:
as in previous comment, maybe more specific `GraphOptimizationApplication` would be better.